pkg/testmachinery/types.go

```diff
@@ -24,7 +24,10 @@ import (
 
 const (
 	// TM_KUBECONFIG_PATH is the path where kubeconfigs are mounted to tests.
-	TM_KUBECONFIG_PATH = "/tmp/env/kubeconfig"
+	TM_KUBECONFIG_PATH = "/tmp/tm/kubeconfig"
+
+	// TM_KUBECONFIG_PATH is the path where kubeconfigs are mounted to tests.
```

Comment:
should be an own comment (copy/paste) :)

Reply:
Done